x509/build/build.rs

```diff
@@ -87,3 +88,33 @@ fn gen_fmc_alias_cert(out_dir: &str) {
     let template = bldr.tbs_template("Caliptra FMC Alias", "Caliptra LDevID");
     CodeGen::gen_code("FmcAliasCertTbs", template, out_dir);
 }
+
+fn gen_rt_alias_cert(out_dir: &str) {
+    let mut usage = KeyUsage::default();
```

Comment:
I think we should include `digitalSignature` and `keyCertSign`.

Reply:
@jhand2 if I am understanding correctly, this build.rs file is just test code, correct? So you are suggesting that the test code should also produce as output the digitalSignature of the CERT. And then something you are calling keyCertSign? What do you mean by keyCertSign? My apologies if I am not correctly following what you meant.

Reply:
build.rs creates a template, and then ROM/FMC fill in anything at runtime that is variable (like TcbInfo fwids).

KeyUsage is an X.509 extension that says what the certificate can be used for. KeyUsage is essentially a bitfield, and two of those bits are:

* keyCertSign: This key can be used to sign other X.509 certificates
* digitalSignature: This key can be used to sign non-certificate data

Depending on the outcome of the quoting discussion, the RT cert may only need the keyCertSign bit set.

These attributes don't change at runtime, so they should be set during template creation.

Reply:
Done